regression/regression_test_enkf.sh

```diff
@@ -35,16 +35,30 @@ maxtime=1200
 # Copy stdout and incr files 
 # from $savdir to $tmpdir
 list="$exp1 $exp2 $exp3"
-for exp in $list; do
-   $ncp $savdir/$exp/stdout ./stdout.$exp
-   nmem=10
-   imem=1
-   while [[ $imem -le $nmem ]]; do
-      member="_mem"`printf %03i $imem`
-      $ncp $savdir/$exp/incr_${global_adate}_fhr06$member $tmpdir/incr$member.$exp
-      (( imem = $imem + 1 ))
+if [[ `expr substr $exp1 1 4` = "rrfs" ]]; then
```

Comment:
Suggest to replace `` with $( )

Reply:
Or does it mean `"${exp1:0:4}"` (i.e. to get the first 4 characters?)

Reply:
done